### ISSUE TITLE
Apply for Temporary... page has form control missing descriptive label and form control missing instructions

### ISSUE THREAD

**Issue by JohnHewittCriterion** [flexion/fs-intake-module#822]
Steps To Verify With JAWS:
 
Launch the website in the Firefox browser.
Move to the Apply for Temporary Outfitter's Permit page at the following URL: https://fs-intake-staging.app.cloud.gov/applications/temp-outfitters/new
Begin completing the form. Check the "Nonprofit" radio button.
Press the Tab key to move to the next field. JAWS announces 'Choose file frame Required'. User does not understand what data is requested for this field.
Press the Up Arrow repeatedly to find the statement "Please attach a copy of your IRS Form 990". This label needs to be associated with the Edit field allowing user to understand what data is requested when the field gains focus.
Press the Tab key to move to " Are you a small entity? check box not checked". User does not understand if checking the checkbox means "Yes, I am a small entity" or "No, I am not a small entity". This field needs instructions to clarify.

WCAG: 3.3 Input Assistance - 3.3.2 Labels or Instructions 
EPERM-96

Tasks
- [ ] Add better labels (John, Seth & Chris)

**Comment by npage123** [comment]
- [ ] Check with Mac Voice Over
- [ ] Check with JAWS
